src/Hook/File/Action/Check.php

```diff
@@ -92,6 +92,7 @@ public function execute(Config $config, IO $io, Repository $repository, Config\A
      * Setup the action, reading and validating all config settings
      *
      * @param \CaptainHook\App\Config\Options $options
+     * @codeCoverageIgnore
```

Comment:
This method does nothing here. It only has a comment in it, but it was showing up in coverage reports as an uncovered method.